.github/workflows/mainnet.yml

```diff
@@ -0,0 +1,56 @@
+name: '🔎 Mainnet Example'
+
+on:
+  schedule:
+    - cron: "0 * * * *"  # Every hour
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+# TODO: Use variables when GH supports it for forks. See https://github.com/orgs/community/discussions/44322
+env:
+  RPC_PROVIDER_URL: "https://mainnet.infura.io/v3/3747007a284045d483c342fb39889a30"
```

Comment:
Maybe this was discussed in the past, but can't we use secrets for this RPC endpoint also? What if someone takes this API key and uses it until the free requests are run out?

Reply:
I agree this is problematic, but so far we have gotten away with this. If move this variable to `secrets`, it's going to break these checks for fork-PR workflow we are currently using.

We can always switch to public RPCs if this becomes an issue.

Reply:
Just as a side note: this endpoint restricts the contract that calls that can be made to.